pkg/provision/provision.go

```diff
@@ -246,6 +246,10 @@ CRIO_MINIKUBE_OPTIONS='{{ range .EngineOptions.InsecureRegistry }}--insecure-reg
 		return err
 	}
 
+	if _, err = p.SSHCommand("sudo systemctl restart crio"); err != nil {
```

Comment:
This seems unrelated to the change. Can it be split out?

Reply:
done